### ISSUE TITLE
1-01-0x Remove spatial extent from variable name

### ISSUE THREAD

**Issue by joergklausen** [wmo-im/wmds#268]
_Please consult the [Guidance](https://github.com/wmo-im/wmds/wiki/How-to-address-problems-and-suggest-changes-to-the-Codetables) and use the following structure for new issues._

**Branch**
[add when created]

**Summary and Purpose**
Several variables contain qualifiers like 'profile' or 'total column' or 'surface' in their name. In general, this context should not be captured in the variable name.  

**Stakeholder(s)**
[include emails or handles as relevant]

**Proposal**
Establish a list of variables concerned and supersede variable names that have the context in their name with those that don't. Other attributes describing the observation such as geometry of observation and observed layer are more adequate to specify this context.

**Reason**
The code lists are simplified if the geometry is not part of the variable name and inconsistent combinations such as 'atmospheric temperature profile' with geometry 'point' can be avoided. Users can specify the geometry explicitly.

**Comment by rhornbrook** [comment]
strongly agree.

**Comment by ejwelton** [comment]
I also agree, relates problems outlines in issue #263 also.

**Comment by fstuerzl** [comment]
Variables with qualifiers refering to "geometry" or "observed layer":

|notation|name| action |
|---|---|---|
|729|Aerosol volcanic ash (Total column)| supersede by new entry "Aerosol volcanic ash", specify geometry: "Total column"|
|328|Cloud ice (total column)|supersede by existing entry 327, "Cloud ice", specify geometry: "Total column"|
|374|Cloud liquid water (CLW, total column)|supersede by existing entry 373, "Cloud liquid water (CLW)", specify geometry: "Total column"|
|262| Surface ozone| supersede by new entry "O3 (ozone)", specify observed layer: "Surface"|
|263| Total column ozone| supersede by new entry "O3 (ozone)", specify geometry: "Total column"|
|264| Vertical ozone profile| supersede by new entry "O3 (ozone)", specify geometry: "Vertical profile"|
|227| Temperature profile| supersede by new entry "Temperature", specify geometry: "Vertical profile"|
|256| Watervapor profile| supersede by new entry "Water vapor", specify geometry: "Vertical profile"|

**Comment by joergklausen** [comment]
The constraint '3D field of' should also disappear, and the appropriate geometry of observation (point, volume) should be attributed instead.

**Comment by fstuerzl** [comment]
All instances of "3D field" removed: https://github.com/wmo-im/wmds/commit/698f6fd779f4637fe73e6a6cb1fc63b92cd88f2f#diff-160fbee712206a9a655a755c35b0b78b48ab1514bf8937984c6069c5fdf6ba6d

**Comment by gaochen-larc** [comment]
I endorse this proposal.

**Comment by joergklausen** [comment]
Regarding
|262 | Surface ozone | supersede by new entry "O3 (ozone)", specify observed layer: "Surface"|
|-- | -- | --|

the observed layer is a concept that is unrelated to geometry, which is what needs to be specified here, i.e.
|262 | Surface ozone | supersede by new entry "O3 (ozone)", specify geometry: "Point"|
|-- | -- | --|

**Comment by joergklausen** [comment]
@ferrighi @jbianchi Please confirm validity of branch.

**Comment by jbianchi81** [comment]
I confirm validity of the branch

**Comment by jbianchi81** [comment]
Sorry that I didn't notice before, but I saw that there is the same issue in 1-01-05. For example, we have:
12009 	Glacier mass balance at a point
12010 	Glacier-wide mass balance
Also, some descriptions use the word 'map', meaning spatial (2D) distribution of the variable:
168 	Lake level 	Map of the height of the lake surface.
237 	Land surface topography	Map of land surface heights.

If you agree, I can make a revision of existing terms in 1-01-05 and provide a proposal. After that I can open a new issue to add new terms

**Comment by jbianchi81** [comment]
List of proposed changes to 1-01-05 (on existing entries):
To merge "land surface temperature" with "soil temperature" (discussion needed)
To change "lake surface temperature" to "water body temperature" (water body could be a term encompassing lakes, rivers and other surface water bodies)
To Merge "lake level" with "river stage (level above reference)" and change description to remove spatial extent
Change "land surface topography" to "land surface height" and change description to remove spatial extent
To merge "soil moisture at roots region" with " soil moisture at surface" 
Change "glacier topography" to "glacier height" and change description to remove spatial extent
Change "ice sheet topography" to "ice sheet height" and change description to remove spatial extent
Merge 2 "glacier mass-balance" entries and remove spatial extent from description
Change "Surface accumulation at a point" to "Glacier ice/snow accumulation" and change description to remove spatial extent
Change "Surface ablation at a point" to "Glacier ice/snow ablation" and change description to remove spatial extent
[1-01-05 revised.csv](https://github.com/wmo-im/wmds/files/6593999/1-01-05.revised.csv)

**Comment by rodicanitu** [comment]
@jbianchi81 - I recommend that we run the proposed changes through the relevant communities, as, for example, the glacier variables have been agreed to, through a consultation with relevant experts, and the same process is under way with sea ice variables.
The particular spatial extent for glaciers, i.e. "at a point" vs "glacier wide" need to be available for selection. not sure this is hte case.
Also, I'm not so sure of the proposal for changing "Surface accumulation at a point" to "Glacier ice/snow accumulation" 
 "Surface ablation at a point" to "Glacier ice/snow ablation". Could you please elaborate on the benefits of collating snow and ice (only glaciers?). I recommend retaining snow and glaciers as separate components of the cryosphere.

**Comment by jbianchi81** [comment]
@rodicanitu thanks for the comments. I agree that the changes run through relevant communities. My intention with this proposal was to remove spatial extent from variable name. But from now on I will propose changes only for hydrology terms.

**Comment by ferrighi** [comment]
The compiled branch including changes in 1-01-01 from @fstuerzl is good for me. I have found only once inconsistency. "Watervapor profile" should be superseded by new entry "Water vapor", but the name used in the branch is "Watervapor" instead. 
@jbianchi81 I agree with @rodicanitu for the table 1-01-05 we should maybe have some expert review this as the changes you are suggesting are not minor?

**Comment by gaochen-larc** [comment]
@joergklausen "Surface Ozone" often refers to ozone measured at 2 m above the ground for monitoring stations.  "specify geometry: "Point"" would need to define the location of the "point", especially when considering aircraft based measurements.

**Comment by fstuerzl** [comment]
> The compiled branch including changes in 1-01-01 from @fstuerzl is good for me. I have found only once inconsistency. "Watervapor profile" should be superseded by new entry "Water vapor", but the name used in the branch is "Watervapor" instead.

@ferrighi, I corrected this in the branch, using the british english version "Water vapour" to be consistent with other entries. 


> @joergklausen "Surface Ozone" often refers to ozone measured at 2 m above the ground for monitoring stations. "specify geometry: "Point"" would need to define the location of the "point", especially when considering aircraft based measurements.

@gaochen-larc, so you suggest to keep the entry for now until we have a way to specify the observed layer?

**Comment by gaochen-larc** [comment]
I think we need to keep "point" as it will be useful for mobile measurements.  I agree with @joergklausen 's idea, but wondering if we can just use "AboveSurface" and define it as measurement from ground level platform at a fixed height(s) above local, e.g., ground site, vehicle based lab, or tower.  Just a thought...

**Comment by ferrighi** [comment]
It seems here the only open discussion is Ozone and for this issue we have a need to specify a geometry. Do we agree to use "O3 (ozone)" as name and "Point" for specifying geometry @gaochen-larc @joergklausen ?

**Comment by joergklausen** [comment]
> It seems here the only open discussion is Ozone and for this issue we have a need to specify a geometry. Do we agree to use "O3 (ozone)" as name and "Point" for specifying geometry @gaochen-larc @joergklausen ?

I would do so. I will also explore if 'observedLayer' could just be another 'parameter' in the OM_Observation ... it may be useful for other observations, as well (e.g., lake _surface_ temperature). Now that we are considering using this feature of the model, we might as well make good use of it. The [heightAboveLocalReferenceSurface](https://schemas.wmo.int/wmdr/1.0/html/) can remain as a detailed specification.

**Comment by fierz** [comment]
@joergklausen @rodicanitu @jbianchi81 et al.
### Glacier variables
In principle I agree with the idea of removing spatial extent from variable names and keeping basic variables only. Thus at first sight, and from my very personal point of view, the two variables _Glacier mass balance at a point_ and _Glacier-wide mass balance_ could be merged to one and be characterized by _point_ and _area_. However, as already suggested by Rodica, I would **strongly advocate** that such a merge/change would have to be discussed by glaciologists first. Indeed, there are subtle differences between point, glaciological, and geodetic mass balance measurements that need to be taken in account. There is currently a task team working on best practices for glaciers that could provide an answer to it. The same is true for changes from _topography_ to _height_. I think the better term would be _thickness_, but again, ask the experts!

### Surface
While I also think an attribute like _observedLayer_ could be helpful in some cases, I strongly disagree with defining surface as a layer. If the definition of a variable says _at its surface_, it does not refer to the geometry of the measurement but to a reference level. In my view this is also true, for example, in the case of snow, lake, or sea ice surface temperature.

**Comment by gaochen-larc** [comment]
> > It seems here the only open discussion is Ozone and for this issue we have a need to specify a geometry. Do we agree to use "O3 (ozone)" as name and "Point" for specifying geometry @gaochen-larc @joergklausen ?
> 
> I would do so. I will also explore if 'observedLayer' could just be another 'parameter' in the OM_Observation ... it may be useful for other observations, as well (e.g., lake _surface_ temperature). Now that we are considering using this feature of the model, we might as well make good use of it. The [heightAboveLocalReferenceSurface](https://schemas.wmo.int/wmdr/1.0/html/) can remain as a detailed specification.

I agree

**Comment by rodicanitu** [comment]
@joergklausen @jbianchi81 @fierz 
after consultations with glaciologists, on variables 12009 Glacier mass balance at a point and 12010 Glacier-wide mass balance, there is wide consensus that 
- Therefore, the glacier experts are advising against removing the spatial extent and changing the variable name. Furthermore, such a change would need a thorough review and broad acceptance in the glaciological community which is a difficult and long process.
- for these two glaciological variables, significant ambiguity would be caused when NOT prominently specifying "point" or "glacier-wide" in the variable name.
- The two variables have quite different implications, regarding their measurement, their further evaluation, their role as climate indicator, as well as for assessing climate change impacts. In fact, the concept of very clearing separating between the two variables has only emerged relatively recently, and there was confusion sometimes before. 

(credit for the summary of feedback: Prof. Matthias Huss, ETH)

In the same spirit, the other glacier variables need to remain as recently recommended in consultation with the relevant expert community. thanks.

**Comment by fierz** [comment]
> In the same spirit, the other glacier variables need to remain as recently recommended in consultation with the relevant expert community. thanks.

Thanks to Mathias Huss for his clear answer on that topic. I thus fully concur with Rodica's conclusion.

**Comment by joergklausen** [comment]
glacier variables remain for the time being, other changes reviewed in branch.

**Comment by amilan17** [comment]
@fstuerzl Can you update the summary of the issue with the details of what is changed and what is superseded?

**Comment by fstuerzl** [comment]
@amilan17, I've updated the proposal above and included a table with all new variables and changed descriptions.
